### ISSUE TITLE
incompatible with curios

### ISSUE THREAD

**Issue by PhiCecHey** [Shadows-of-Fire/Placebo#28]

**Comment by Shadows-of-Fire** [comment]
This is not a valid report by any means, apoth uses curios alongside placebo 4.6, and you have't provided any actual data

**Comment by PhiCecHey** [comment]
how come you think that apotheosis uses curios? curios is not listed in the relations section of apotheosis.

**Comment by PhiCecHey** [comment]
ok, sorry, let me rephrase:
placebo 4.6.0 does not run.
